src/features/game/lib/constants.ts

```diff
@@ -468,6 +468,7 @@ export const INITIAL_FARM: GameState = {
   inventory: {
     Sunflower: new Decimal(5),
     Potato: new Decimal(12),
+    Carrot: new Decimal("502.079999999999991"),
```

Comment:
did you have tried out to use pure 0.9 floating points? i.e. `502.999999999999999` it's 15 - 9 right there

Reply:
works just fine. Only the modal shows rounded up, but the actual value is the one sold.

https://user-images.githubusercontent.com/94913189/196141315-ed9dc3a8-f93d-47f9-a4b7-63cd0fe5a769.mp4